sdk/testing/stepwise/stepwise_test.go

```diff
@@ -0,0 +1,282 @@
+package stepwise
```

Comment:
I'm seeing a 63% code coverage of the `stepwise.go` file - mostly on the happy path. Are you planning on adding more tests for this in subsequent PRs?

Reply:
Yes. As the first environment gets added and the first few actual tests to use the framework arrive, I plan to expand on the test coverage